projectq/ops/_relative_command.py

```diff
@@ -0,0 +1,69 @@
+from projectq.ops._metagates import ControlledGate
+
+class RelativeCommand(object):
+    """ 
+    Class:
+        Used to represent commands when there is no engine.
+        i.e. in the definition of a relative commutable_circuit 
+        within a gate class. 
+
+    Attributes:
+        gate: The gate class.
+        _gate: If RelativeCommand.gate = ControlledGate, _gate will
+        return the gate class on the target qubit. 
+        e.g. if relative_command.gate = CNOT 
+        (class = projectq.ops._metagates.ControlledGate)
+        relative_command._gate = NOT
+        (class = projectq.ops._gates.XGate)
```

Comment:
I would simplify the docstrig for `_gate` by simply saying that it is the true gate class if `gate` is  a metagate.